logging/config/cluster-logging-external.adoc

```diff
@@ -1,26 +1,25 @@
 :context: cluster-logging-external
 [id="cluster-logging-external"]
-= Sending {product-title} logs to external devices
+= Sending logs to external devices using Fluentd plugins
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can send Elasticsearch logs to external devices, such as an externally-hosted Elasticsearch instance
-or an external syslog server. You can also configure Fluentd to send logs to an external log aggregator.
+{product-title} cluster logging allows you to configure Fluentd plugins to send logs to external devices. 
 
-[NOTE]
-====
-You must set cluster logging to Unmanaged state before performing these configurations, unless otherwise noted.
-For more information, see xref:../../logging/config/cluster-logging-management.adoc#cluster-logging-management[Changing the cluster logging management state].
-====
+* *out_foward*. You can use this plug in to send logs to an external log aggregator.
+* *fluent-plugin-remote-syslog*. You can use this plug in to send logs to external syslog server.
+
+Alternatively, you can use the xref:../../logging/config/cluster-logging-log-forwarding.adoc#cluster-logging-log-forwarding[Log Forwarding feature] that can be easier to configure than the plugins. Note that Log Forwardinging is currently in Tech Preview.
```

Comment:
Should this Note about log forwarding be a proper `[NOTE]` ?

Reply:
s/Tech Preview/Technology Preview